Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.9-slim-buster
 
 RUN  apt-get update && \
   apt-get install -y procps && \
+  apt-get install -y gcc && \
+  apt-get install -y g++ && \  
```

Comment:
I was able to get the docker container to build on my local machine by adding these two lines. The underlying error that I was getting was `error: command 'gcc' failed: No such file or directory`. [This](https://github.com/watson-developer-cloud/python-sdk/issues/418#issuecomment-674038596) GitHub thread provided this solution for me.

Reply:
Weird. now I'm struggling to install this locally, but if it works for you, then i'll approve!